assets/css/footer.css

```diff
@@ -47,3 +29,8 @@ body {
 #version:hover, #version:active, #version:focus {
     text-decoration: none;
 }
+
+.helpMessage {
+    border-radius: 3px;
```

Comment:
I think this element already has a `border-radius` set due to `.card`. Does it need to be set again?

Reply:
Yup. :)